zeekpkg/manager.py

```diff
@@ -231,7 +231,7 @@ class Manager(object):
             load all installed packages that have been marked as loaded.
     """
 
-    def __init__(self, state_dir, script_dir, plugin_dir, zeek_dist='',
+    def __init__(self, state_dir, script_dir, plugin_dir, cmake_dir, zeek_dist='',
```

Comment:
We're using `zeekpkg.Manager` and this is an API change in that `cmake_dir` is now required to construct a `Manager`. I haven't looked at the rest, but can it be defaulted to an empty string? In the environment I'm having in mind, there's also no reasonable `cmake_dir` available to be provided.

Would also suggest to add it after the last parameter, otherwise the API changes when using positional arguments only. Though I really hope keyword args are used for construction elsewhere.

Reply:
I think this patch is not the correct fix, and we will not add it in this form.

Reply:
Alright, thanks :-)